drivers/ethernet/Kconfig.nxp_enet

```diff
@@ -12,7 +12,7 @@ config ETH_NXP_ENET
 	select ARM_MPU if CPU_CORTEX_M7
```

Comment:
Typo in commit msg of commit `net: dsa: make current DSA core driver obsolete`
s/deivices/devices/
Perhaps the sentence could say:
`Once the legacy devices are converted to new DSA framework, this code could be removed`

Reply:
Thanks, will fix.